dist/index.html

```diff
@@ -1,10 +1,15 @@
 <!DOCTYPE html>
-  <html>
-    <head>
-      <meta charset="utf-8" />
-      <title>Leap of Faith</title>
-    </head>
-    <body>
-      <script src="main.js"></script>
-    </body>
-  </html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Leap of Faith</title>
+  <link rel="stylesheet" href="../src/css/index.css">
```

Comment:
You won't need to manually import the CSS files here, WebPack does this for you.
Check `9a9db789d763cd56a0f7fc83f816ad0b09591c05`